### PR TITLE
Fix naming of helm releases

### PR DESCRIPTION
**Pull request by philipp94831**

### CODE REVIEW

**Review by disrupted — COMMENTED** [review]
do we need to update any instructions for the default Helm usage (without KPOps) now that the `-clean` suffix isn't appended automatically anymore?

**Comment by philipp94831** [comment]
> do we need to update any instructions for the default Helm usage (without KPOps) now that the `-clean` suffix isn't appended automatically anymore?

I don't think so. Job will now have the name of the helm release but everything still should work

File: charts/producer-app-cleanup-job/templates/_helpers.tpl [hunk 1]
```diff
@@ -17,7 +17,7 @@ If release name contains chart name it will be used as a full name.
 {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
 {{- else -}}
 {{- $name := default .Values.nameOverride -}}
-{{- printf "%s-clean" $name | trunc 63 | trimSuffix "-" -}}
+{{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
```
Comment (disrupted) [inline charts/producer-app-cleanup-job/templates/_helpers.tpl:20]: probably the formatstring is not needed anymore and there's a better way to write it, but I am fine with it

**Review by disrupted — APPROVED** [review]

File: charts/producer-app-cleanup-job/templates/_helpers.tpl (hunk 1 above)
Comment (philipp94831) [reply charts/producer-app-cleanup-job/templates/_helpers.tpl:20]: I copied it from other charts in this repo. You are probably right but I would keep it for now

**Pull request merged**